src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java

```diff
@@ -59,7 +59,8 @@
  *   <tr><td>ordered</td><td>whether imports within group should be sorted</td>
  *       <td>Boolean</td><td>true</td></tr>
  *   <tr><td>separated</td><td>whether imports groups should be separated by, at least,
- *       one blank line and aren't separated internally</td><td>Boolean</td><td>false</td></tr>
+ *       one blank line or command and aren't separated internally
```

Comment:
what is `command`? The only thing allowed between imports is more imports and comments. I'm not sure if annotations can be added.

Reply:
sorry for my carelessness.it is 'comment'.